src/status_im2/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -0,0 +1,62 @@
+(ns status-im2.contexts.wallet.send.input-amount.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.contexts.wallet.send.input-amount.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- view-internal
+  [_]
+  (let [token                 :eth
+        {:keys [currency]}    (rf/sub [:profile/profile])
+        networks              (rf/sub [:wallet/network-details])
+        limit                 286.32
+        limit-label           (str limit " " (string/upper-case (name token)))
+        input-title           (i18n/label :t/send-limit {:limit limit-label})
+        input-value           (reagent/atom "")
+        conversion-rate       1800
+        handle-keyboard-press (fn [v]
+                                (let [new-value (str @input-value v)
+                                      num-value (parse-double new-value)]
+                                  (when (<= num-value limit)
+                                    (reset! input-value new-value))))
+        handle-delete         (fn [_]
+                                (swap! input-value #(subs % 0 (dec (count %)))))]
+    (fn [{:keys [on-confirm]
+          :or   {on-confirm #(js/alert "Confirmed")}}]
+      [rn/view
+       {:style style/screen}
+       [quo/page-nav
+        {:background       :blur
+         :icon-name        :i/arrow-left
+         :on-press         #(rf/dispatch [:navigate-back])
+         :right-side       :account-switcher
+         :account-switcher {:customization-color :yellow
+                            :emoji               "🎮"
+                            :on-press            #(js/alert "Switch account")}}]
+       [quo/token-input
+        {:container-style style/input-container
+         :token           token
+         :currency        currency
+         :networks        networks
+         :title           input-title
+         :conversion      conversion-rate
+         :show-keyboard?  false
+         :value           @input-value}]
+       [rn/scroll-view]
```

Comment:
what is that empty `scroll-view` ?

Reply:
This is container for other components on this page

Reply:
yeah but it needs to wrap those components, no? 🤔

Reply:
<img width="431" alt="Screenshot 2023-11-06 at 13 14 59" src="https://github.com/status-im/status-mobile/assets/10757633/2238509c-de30-485b-bfa5-9d3f01c3284c">
Not yet. =) This container for these components, that will be implemented in another issue.
I added it so that styles of this screen correspond to the design and they do not need to be changed in the future.

Reply:
ah got it, maybe you can add a comment there - ; network routing content to be added

Reply:
Just comment in code? Or visible component inside this block like placeholder?

Reply:
I think the comment in the code is fine for now 👍